Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/swift-server/async-http-client.git", from: "1.1.0"),
-        .package(url: "https://github.com/apple/swift-crypto.git", "1.0.0" ..< "3.0.0"),
+        .package(url: "https://github.com/apple/swift-crypto.git", "1.0.0" ..< "4.0.0"),
```

Comment:
Given the JWTKit limitation on the _minimum_ version, would it make more sense to just do `from: "3.0.0"` here? The only reason most Vapor packages don't do that is to try to remain compatible with other dependencies with the same issue.

Reply:
Given that it might prevent some people to get the latest releases of stripe-kit (e.g. a dependency set to `from: "2.0.0"` in a `Package.swift` file), I guess keeping the extended range would be more appropriate?
I’m ok for both solutions though.